test/testsuite.h

```diff
@@ -99,8 +99,8 @@ extern std::ostringstream errout;
 extern std::ostringstream output;
 
 #define TEST_CASE( NAME )  if ( prepareTest(#NAME) ) { setVerbose(false); NAME(); }
-#define ASSERT( CONDITION )  if (!assert_(__FILE__, __LINE__, CONDITION)) return
-#define ASSERT_EQUALS( EXPECTED , ACTUAL )  if (!assertEquals(__FILE__, __LINE__, EXPECTED, ACTUAL)) return
+#define ASSERT( CONDITION )  if (!assert_(__FILE__, __LINE__, (CONDITION))) return
+#define ASSERT_EQUALS( EXPECTED , ACTUAL )  if (!assertEquals(__FILE__, __LINE__, (EXPECTED), (ACTUAL))) return
```

Comment:
ran into a few problems with arguments containing commas.

Reply:
ok strange.. well this looks better.